src/SimpleExcelReader.php

```diff
@@ -70,6 +72,7 @@ public function getReader(): ReaderInterface
     public function skip(int $count): SimpleExcelReader
     {
         $this->skip = $count;
```

Comment:
Can't we make `skip` nullable. That way `null` could signify that skip shouldn't be used. This way we can lose the `useSkip` instance variable.

Reply:
Actually, leaving it as 0 would do too, it would return false in a statement.

Reply:
I followed Francismori7's advice below and removed useSkip, as freek wanted